website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Quickstart"
+id: quickstart-semantic-layer
+description: "Define metrics and set up the dbt Semantic Layer"
+sidebar_label: "Quickstart"
+---
+
+# dbt Semantic Layer quickstart
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
```

Comment:
Suggestion: Should say "in Public Preview."

We should make all instances of the term “Public Preview” be in title case, with capital P’s. This is a proper noun referring to a specific product state with specific definitions unique to dbt Labs’ product life cycle, not a common noun.